unity_cogs/embedremover.py

```diff
@@ -0,0 +1,33 @@
+import discord
+from discord.ext import commands
+from unity_util import bot_config
+import logging
+import sys
+import random
+
+logging.basicConfig(level=logging.INFO, format="[%(levelname)s]\t %(name)s: %(message)s", handlers=[
+    logging.StreamHandler(sys.stdout),
+    logging.FileHandler(f'./logs/{bot_config.LOGGING_FILENAME}')
+])
+
+
+class EmbedRemover(commands.Cog):
+    def __init__(self, client):
+        self.client = client
+
+    # Events
+    @commands.Cog.listener()
+    async def on_ready(self):
+        print('Renderdistance meme cog online')

```

Comment:
Make sure the log message is correct. Eventually I will break this out into a utility decorator which does this automatically, but for now please change it.

Reply:
Also, use `logging.info`